src/i18n.js

```diff
@@ -0,0 +1,82 @@
+import i18n from "i18next";
```

Comment:
I think we don't need `i18next` and `react-i18next`
we can defined a `language.js` file 

```js
const languages = {
   en : {
       name : 'xxx'
    }
   zh : {
      name: 'xxx'
   }
}
```

in music player component 

```js
const language = languages[this.props.lng]
language.name // xxx
```
Do you think so ?
first, thank you very much for your pr
now there's a little bit too much dependency

Reply:
Thank you for your great component. I'm pleased to contribute to your greate work.
I'm a newby in React and I'm digging in about internationalization of a project of my own. I'm trying to find the best practices to do that and I'll apply to my contribution to your project.